hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/schema/TablesStorage.java

```diff
@@ -44,12 +47,21 @@
 import java.util.concurrent.CompletableFuture;
 import java.util.stream.Collectors;
 
+import static com.hazelcast.config.MapConfig.INFINITE_TTL_SECONDS;
 import static java.util.stream.Collectors.toList;
 import static java.util.stream.Collectors.toSet;
 
 public class TablesStorage {
     static final String CATALOG_MAP_NAME = "__sql.catalog";
 
+    static final MapConfig SQL_CATALOG_MAP_CONFIG = new MapConfig()
+            .setName(CATALOG_MAP_NAME)
+            .setBackupCount(MapConfig.MAX_BACKUP_COUNT)
+            .setTimeToLiveSeconds(INFINITE_TTL_SECONDS)
+            .setReadBackupData(true)
+            .setMergePolicyConfig(new MergePolicyConfig().setPolicy(DiscardMergePolicy.class.getName()))
```

Comment:
We should not discard changes in the split members, but use the more recent version. Due to the low frequency of DDL it's fair to assume that the clocks are synchronized sufficiently.
```suggestion
            .setMergePolicyConfig(new MergePolicyConfig().setPolicy(LatestUpdateMergePolicy.class.getName()))
```